materialize-styles.loader.js

```diff
@@ -7,6 +7,7 @@ module.exports = function(content) {
     var config = this.exec(content, this.resourcePath);
     var start =
             "$roboto-font-path: \"~materialize-css/fonts/roboto/\";\n"
+            + "@import \"~materialize-css/sass/components/_color.scss\";\n";
```

Comment:
True fact, that must be the reason why I didn't include it at the beginning @jasonzhu1991.

What is the best solution, being able to use functions like ```lighten``` which is contained in the color file or being able to customize colors ?
But we can't get both.